shipctl/x86_64/Ubuntu_16.04/utility.sh

```diff
@@ -689,6 +689,70 @@ put_resource_state_multi() {
   done
 }
 
+split_tests() {
+  if [ "$1" == "" ] || [ "$2" == "" ] || [ "$3" == "" ]; then
+    echo "Usage: shipctl split_tests TEST_PATH TEST_FILES_NAME_REGEX TEST_REPORTS_PATH"
+    exit 99
+  fi
+
+  # delete tmp files
+  rm -rf /tmp/current_tests.txt
+  rm -rf /tmp/past_test_timings.txt
+  rm -rf /tmp/sorted_past_test_timings.txt
+  rm -rf /tmp/sorted_past_tests.txt
+
+  # create tmp files
+  touch /tmp/current_tests.txt
+  touch /tmp/past_test_timings.txt
+  touch /tmp/sorted_past_test_timings.txt
+  touch /tmp/sorted_past_tests.txt
+
+  for i in "$(find $1 -name $2)";
+  do
+    echo -e "$i\n" >> /tmp/current_tests.txt;
+  done
+
+  find $3 -name \*.xml|while read i;
+  do
+    echo $(xq .testsuites.testsuite $i | jq -cr '.["@filepath"]," ", .["@time"]') >> /tmp/past_test_timings.txt
+  done
+
+  IFS=$'\r\n' GLOBIGNORE='*' command eval  'past_tests=($(cat /tmp/sorted_past_tests.txt))'
```

Comment:
You're missing something. This file was never written to.

Reply:
Added the missing commands